client/my-sites/upgrades/domain-management/list/index.jsx

```diff
@@ -67,6 +76,19 @@ const List = React.createClass( {
 					sitePlans={ this.props.sitePlans } />
 				{ this.domainWarnings() }
 
+				{ this.props.hasDomainCredit && <Notice
```

Comment:
This is silly thing, but I prefer to have the beginning of the component element in a new line for visual scanning.